docs/dev/developer-guides/bridging/l1-l2-interop.md

```diff
@@ -1,72 +1,42 @@
-
-### How it works in zkSync Lite
-
-In the previous version of zkSync, we only had two operations that could be sent to zkSync from L1:
-
-- `Deposit` to bridge funds from Ethereum to zkSync.
-- `FullExit` to bridges the funds back from Ethereum (this is essentially the same as `Withdraw` in zkSync Era).
-
-If users wanted to deposit funds to or withdraw funds from zkSync, they would have to send a transaction request to the smart contract which will then get appended to the queue of priority transactions. The queue has the following rules:
-
-- All transactions are processed sequentially.
-- Each priority operation must be processed by the operator within `X` days since it was submitted to the contract.
-
-The first rule is strictly enforced by the smart contract. The second rule may be violated if the operator becomes malicious or unavailable. In case that happens, the system enters 'exodus mode', where no new blocks can be processed and users can withdraw their funds without cooperation from the operator.
+## L1 to L2 communication
 
-### What changes are needed?
+L1 to L2 communication is governed by the [`IZkSync.sol`](https://github.com/matter-labs/v2-testnet-contracts/blob/b8449bf9c819098cc8bfee0549ff5094456be51d/l1/contracts/zksync/interfaces/IZkSync.sol#L4) inherited interfaces.
 
-The process described above works well for a system with a small set of relatively light supported operations. zkSync Era supports general smart contract computation, and thus some principles had to be changed in order to preserve the stability of the network.
+:::tip
+- If you prefer to learn-by-doing, the [cross chain governance tutorial](../../tutorials/cross-chain-tutorial.md) is a practical example of layer interoperability.
+:::
 
-Firstly, all transactions need to be supported by the priority queue. Users may have their funds locked on an L2 smart contract, and not on their own L2 account. Therefore before moving their funds to L1, they need to send an `Execute` transaction to the zkSync network to release the funds from that smart contract first.
+### Gas estimation
 
-Secondly, the priority queue needs to stay censorship-resistant. But imagine what will happen if users start sending a lot of transactions that take the entirety of the block gas limit? There needs to be a way to prevent spam attacks on the system.
-That's why submitting transactions to the priority queue is no longer free.
-Users need to pay a certain fee to the operator for processing their transactions. It is really hard to calculate the accurate fee in a permissionless way.
-Thus, the fee for a transaction is equal to `txBaseCost * gasPrice`. The `gasPrice` is the gas price of the users' transaction, while `txBaseCost` is the base cost for the transaction, which depends on its parameters (e.g. `gas_limit` for `Execute` transaction).
+The system processes gas estimation for transactions implicitly. However, it is also possible to implement the gas estimation processes explicitly.
```

Comment:
Maybe a matter of taste, but I call zkSync "the system", so the system does not calculate the limit implicitly. The SDK does that

```suggestion
The SDK processes gas estimation for transactions implicitly. However, it is also possible to implement the gas estimation processes explicitly.
```